docs/_manage/invite-team-member.md

```diff
@@ -81,3 +81,31 @@ To sign up as a collaborator, log in to Zapier (or sign up for Zapier if you don
 ![JoinIntegrationFromDashbaord](https://cdn.zappy.app/5a3452ea8e72049a2083aab5dac59069.png)
 
 ![JoinIntegration](https://cdn.zappy.app/b47f3a9b672533df505667cf3933a9ee.png)
+
+## Managing Partner Email Subscriptions
+
+### General Partner Marketing Updates
+
+Zapier sends out general partner marketing updates related to your integration and the Zapier platform. These emails include:
+* Monthly Partner Newsletter with integration data insights
+* Outreach about partnership and co-marketing opportunities 
+
+To control your subscription for these emails, visit the [email preferences page](https://developer.zapier.com/partner-settings/email) in the developer dashboard and opt in or out per integration as needed.
+
+### Platform and Partner Program Updates
+
+We'll keep you in the loop with Zapier Platform and Partner Program Updates. These emails include:
+* Alerts about user [Bugs and Feature Requests](https://platform.zapier.com/manage/analyze-integration-performance#bugs)
+* Critical Platform and Program Updates
+
+You cannot unsubscribe from Zapier Platform and Partner Program Updates unless an existing integration team member (with an equal or greater role as you^) removes you from the integration team entirely. These emails deliver essential announcements about the Partner Program, Zapier Platform features, product updates, and compliance with integration quality standards to ensure you're always up-to-date with changes that could impact your use of the Zapier Platform.
+
+> **^An Admin can remove another Admin or Collaborator from the integration team. A Collaborator can only remove other Collaborators from the integration team.**
```

Comment:
Small suggestion, not blocking:

From a quick search in the repo for ">" it looks like we use "> **Note:**" around the docs a lot. Definitely not a set standard, but might be nice to use here versus a carrot! I usually see asteriks for footnotes like this, which maybe you couldn't use here bc of the bold-ing markdown?